[IMP] IoT: edits to all IoT docs (#3673)
odoo/documentation: content/applications/productivity/iot/devices/footswitch.rst

====================
Connect a footswitch
====================

When working, it is always better for an operator to have two hands available. Odoo's
:abbr:`IoT (Internet of Things)` Box makes this possible when using a footswitch.

In fact, the operator will be able to go from one screen to another and perform actions by using
their foot and the footswitch. This can be configured in just a few steps.

Connection
==========

To connect the footswitch to the :abbr:`IoT (Internet of Things)` Box, just connect the two by
cable. More often than not this is a :abbr:`USB (Universal Serial Bus)` cable.

If the footswitch is a `supported device <https://www.odoo.com/page/iot-hardware>`__, there is no

> need to set up anything since it will be automatically detected when connected.

Reviewer: ```suggestion
need to take further action since it will be automatically detected when connected.
```

.. image:: footswitch/footswitch_01.png
   :align: center
   :alt: Footswitch recognized on the IoT box.

> Link a Footswitch to a Workcenter in the Manufacturing App

Reviewer: ```suggestion
Link a footswitch to a workcenter in the manufacturing app
```

Connect a Footswitch
====================

When working, it is always better for an operator to have two hands available. Odoo's
:abbr:`IoT (Internet of Things)` Box makes this possible when using a footswitch.

In fact, the operator will be able to go from one screen to another and perform actions by using

> their foot and the footswitch. This is very convenient, and it can be configured in just a few

Reviewer 2: ```suggestion
their foot and the footswitch. This can be configured in just a few
```
I think the convenience line might be too sales-y

steps.

Connection
==========

Connecting the footswitch to the *IoT Box* is easy, just connect the two by cable. This

> is more often than not a USB cable.

Author: Changed to: More often than not this is a USB cable.

If the footswitch is a `supported device <https://www.odoo.com/page/iot-hardware>`__, there is no
need to set up anything since it will be automatically detected when connected.

.. image:: footswitch/footswitch_01.png
   :align: center
   :alt: Footswitch recognized on the IoT box.

Link a Footswitch to a Workcenter in the Manufacturing App
==========================================================

> To link the footswitch to an action, it needs to be configured on a workcenter. Go to the workcenter

Author: Noted, I've added an example on validate and mark as done in the Work Order.
